scripts/config/msu/text.nut

```diff
@@ -13,4 +13,23 @@
 	{
 		return this.color(::Const.UI.Color.NegativeValue, _string);
 	}
+
+	function colorizeValue( _value, _invert = false, _addSign = true )
+	{
+		if (_value < 0) return this[_invert ? "colorGreen" : "colorRed"](_value);
```

Comment:
Missing - from `_addSign`

Reply:
Negative doesn't need to be added because negative numbers already print with a negative sign in strings.

Reply:
I'm an idiot, nice

Reply:
Actually in that case, I'd *remove* the sign if `_addSign` is false. This would come in useful if you were to say something like "increases Thing by x" and "decreases Thing by x" rather than "increases Thing by -/+ x"